packages/graphql-transformers-e2e-tests/src/__tests__/SubscriptionsWithAuthTest.e2e.test.ts

```diff
@@ -661,12 +682,12 @@ test('Test that an user not in the group is not allowed to view the subscription
         }
       }
     `,
-    fetchPolicy: 'network-only',
-  });
+    authMode: GRAPHQL_AUTH_MODE.AMAZON_COGNITO_USER_POOLS,
+  }) as Observable<any>;
   const subscriptionPromise = new Promise((resolve, _) => {
     observer.subscribe({
       error: (err: any) => {
-        expect(err.errors[0].message).toEqual(
+        expect(err.error.errors[0].message).toEqual(
           'Connection failed: {"errors":[{"errorType":"Unauthorized","message":"Not Authorized to access onCreateStudent on type Subscription"}]}',
         );
         resolve(undefined);
```

Comment:
Can we do something like this when returning the promise from the test so we have some faster timeout
```typescript
return withTimeOut(subscriptionPromise, SUBSCRIPTION_WAIT_TIMEOUT, 'OnCreateStudent subscription timed out');
```
and implement and helper function which does reject like this
```typescript
export function withTimeOut<T>(promiseToWait: Promise<T>, timeout: number, timeoutMessage?: string, cleanupFn?: () => void): Promise<T> {
  const timeoutPromise = new Promise<T>((_, reject) => {
    setTimeout(async () => {
      if (cleanupFn) {
        await cleanupFn();
      }
      reject(new Error(timeoutMessage || 'Waiting timed out'));
    }, timeout);
  });
  return Promise.race([promiseToWait, timeoutPromise]);
}
```